openlibrary/templates/account/books.html

```diff
@@ -100,6 +102,12 @@ <h3>
                 <li><a class="list-link" href="$(lst.key)">$lst['name'] ($(len(lst.seeds)))</a></li>
             </div>
           </ul>
+        $if key == 'want-to-read':
+          <ul class="preset-lists">
+            <li class="subsection">$_('Sort Filters')</li>
+            <li><a href="$ctx.path?sort=descending" $('class=selected' if date_added == 'descending' else '')>$_('Newest to Oldest')</a></li>
+            <li><a href="$ctx.path?sort=ascending" $('class=selected' if date_added == 'ascending' else '')>$_('Oldest to Newest')</a></li>
+          </ul>
```

Comment:
UX-wise, we'll likely want it to look like this, just above the pagination section:

![image](https://user-images.githubusercontent.com/6251786/123495550-300ea480-d5f2-11eb-967c-51e025844442.png)

Something like (with some if's similar to what you have now)

```html
<span class="mytools-tools"><img src="/images/icons/icon_sort.png" alt="$_('Sorting by')" style="margin-right:10px;" width="9" height="11">
    <strong class="lightgreen">$_("Date Added (newest)")</strong>
    |
    <a href="$ctx.path?sort=asc">$_("Date Added (oldest)")</a>
</span>
```